app/views/presentations/index.html.erb

```diff
@@ -23,6 +23,9 @@
   </tbody>
   <tfoot>
     <tr>
+      <td><%= render 'filter' %></td>
```

Comment:
What about the idea of having this filter at the top somewhere. The table will differ in row count making the filter show up at different places down the page. If it is above all the rows it will always been in the same place.